### ISSUE TITLE
Critical severity vulnerability from superagent->formidable

### ISSUE THREAD

**Issue by samarpanB** [pubnub/javascript#278]
There is a critical severity vulnerability showing up on snyk dashboard for pubnub, due to superagent. It can be seen here -> https://snyk.io/test/npm/pubnub

Can we please fix this ?

**Comment by ckotyan** [comment]
Yes, it would be great to fix this as my company's libraries are failing audit due to this vulnerability. [Super agent](https://github.com/ladjs/superagent/pull/1800) folks are already trying to fix this FWIW.

**Comment by tehseenc** [comment]
Bumping this! It came up in an audit on our end as well.

**Comment by parfeon** [comment]
PubNub SDK doesn't use `superagent` anymore.
